common/djangoapps/util/enterprise_helpers.py

```diff
@@ -19,7 +23,7 @@ def enterprise_enabled():
     """
     Determines whether the Enterprise app is installed
     """
-    return 'enterprise' in settings.INSTALLED_APPS
+    return 'enterprise' in settings.INSTALLED_APPS and not FAILED_ENTERPRISE_IMPORT
```

Comment:
I'm running into an odd issue (and help from anyone else would be very welcome) that this was helping me diagnose - basically, Jenkins wasn't installing the updated version of enterprise, but using the existing version.

Now, after a few tweaks, it's installing the updated version, but the tests that use it are acting as though the import fails.

Regardless, I've pulled this item, in the hope that I'll get something more informative.